site/gamedata.php

```diff
@@ -137,6 +137,15 @@ function num_in_kilo ( $num ) {
             <section class="mb-5">
                 <!-- Publications -->
                 <h3>Publications</h3>
+                <?php
+                    foreach ($game->getPublications() as $value) {
+                        echo '<div class="card shadow mb-4">
+                            <div class="card-body">
+                                <a class="btn btn-secondary btn-publication" href="' . $value->Link . '">' . $value->StudyName . '</a>
```

Comment:
The values might have characters that need to be encoded for HTML so change `$value->Link` to `htmlspecialchars($value->Link)` and `$value->StudyName` to `htmlspecialchars($value->StudyName)`. I had Michael make similar changes in your parent branch.